### DISCUSSION TITLE
What's the best way to write an In state with multiple columns

### DISCUSSION THREAD

**Discussion by chriskolenko** [uptrace/bun#491] [General]
I have this following query:

```
	if err := tx.NewSelect().
		Table("table").
		Where("(c1, c2) IN (?)", bun.In(ids)).
		Scan(ctx, &out); err != nil {
		return nil, err
	}
	return out, nil
```

ids is a slice of structs. 

Currently this will return an error. Just wanted to know if there was a way to do this type of query?

**Comment by vmihailenco** [comment]
Try `bun.In([][]interface{{1, 2}, {3, 4}, {5, 6}})`
